client-sdk/ts-web/rt/playground/src/index.js

```diff
@@ -0,0 +1,22 @@
+// @ts-check
+
+import * as oasisRT from './../..';
+
+(async function () {
+    try {
```

Comment:
There ends up not being much here, because we went to focus on getting the bridge SDK working. Maybe we could put up a simple-keyvalue client here someday.

Reply:
Yeah once we have an example "simple-keyvalue" runtime that uses the runtime-sdk.